pkg/helm/chartproxy/repos_test.go

```diff
@@ -512,3 +516,158 @@ func TestHelmRepoGetter_SkipDisabled(t *testing.T) {
 		})
 	}
 }
+
+func TestHelmRepoGetter_unmarshallConfig(t *testing.T) {
+	//create the server.key and server.crt
+	//create the server.key and server.crt
+	err := ExecuteScript("./testdata/createTlsSecrets.sh")
+	require.NoError(t, err)
+	//start chartmuseum server
+	err = ExecuteScript("./testdata/chartmuseum.sh")
+	require.NoError(t, err)
+	err = ExecuteScript("./testdata/cacertCreate.sh")
+	require.NoError(t, err)
+	err = ExecuteScript("./testdata/uploadCharts.sh")
```

Comment:
Error handling is missing here.

Reply:
It is done intentionally as we get exit status 7 on the upload chart completion.

Reply:
`uploadCharts.sh` might be missing an `exit 0` at the end after all the operations are successful to indicate success; would help to also add `set -e` to the script so it fails if any other command in the script fails (https://rotadev.com/what-does-set-e-mean-in-a-bash-script-dev/).

With the change above, the errors returned by this call are meaningful and can stop the test at the appropriate step in the test flow.